cartoframes/context.py

```diff
@@ -167,6 +167,19 @@ def read(self, table_name, limit=None, index='cartodb_id',
 
         return self.query(query, decode_geom=decode_geom)
 
+    def tables(self):
+        datasets = DatasetManager(self.auth_client).filter(
+            show_table_size_and_row_count='false',
+            show_table='false',
+            show_stats='false',
+            show_likes='false',
+            show_liked='false',
+            show_permission='false',
+            show_uses_builder_features='false',
+            show_synchronization='false',
+            load_totals='false')
+        return (Table.from_dataset(d) for d in datasets)
```

Comment:
Why did you opt for a generator here? I'm guessing that the most common use case for this method is to get a full list of all the tables and it might not be obvious that this should be iterated over.

Reply:
When languages provide lazy declaration I use it by default, and other people have recently suggested using iterators instead in other similar cases. Anyway, switching to list if you think it's better 👍